### ISSUE TITLE
Deploy application  failes with "Specifying a root certificates file with the insecure flag is not allowed"

### ISSUE THREAD

**Issue by Warden-Thresh** [kubesphere/kubesphere#2435]
## English only!

**注意！GitHub Issue 仅支持英文，中文 Issue 请在 [论坛](https://kubesphere.com.cn/forum/) 提交。**

**General remarks**

> Please delete this section including header before submitting
>
> This form is to report bugs. For general usage questions refer to our Slack channel
>        [KubeSphere-users](https://join.slack.com/t/kubesphere/shared_invite/enQtNTE3MDIxNzUxNzQ0LTdkNTc3OTdmNzdiODViZjViNTU5ZDY3M2I2MzY4MTI4OGZlOTJmMDg5ZTFiMDAwYzNlZDY5NjA0NzZlNDU5NmY)

**Describe the Bug**
When deploy any application 
`
specifying a root certificates file with the insecure flag is not allowed
`

```
2020-07-15 11:16:50.53015 -ERROR- Parse cluster conf with versionId [appv-PqGWz2yjpqB1] runtime [host] conf [replicaCount: 1\nimage:\n  nginx:\n    repository: nginx\n    tag: 1.17.5-alpine\n    pullPolicy: IfNotPresent\nnameOverride: ''\nfullnameOverride: ''\nservice:\n  name: http\n  type: ClusterIP\n  port: 80\ningress:\n  enabled: false\n  annotations: {}\n  paths: []\n  hosts:\n    - nginx.local\n  tls: []\nextraVolumes: []\nextraVolumeMounts: []\nextraInitContainers: []\nreadinessProbe:\n  path: /\n  initialDelaySeconds: 5\n  periodSeconds: 3\n  failureThreshold: 6\nlivenessProbe:\n  path: /\n  initialDelaySeconds: 5\n  periodSeconds: 3\nresources: {}\nconfigurationFile: {}\nextraConfigurationFiles: {}\nnodeSelector: {}\ntolerations: []\naffinity: {}\nName: nginx-fq4xfc\nDescription: ''\nWorkspace: zhny\n] failed: rpc error: code = Unknown desc = specifying a root certificates file with the insecure flag is not allowed (handler.go:673)
```
![image](https://user-images.githubusercontent.com/22620409/87539978-6c8f0c00-c6d1-11ea-8383-1c9564b7681a.png)

**Versions Used**
KubeSphere: 3.0
Kubernetes: (If KubeSphere installer used, you can skip this)

**Environment**
How many nodes and their hardware configuration:

For example: CentOS 7.8 / 1 masters:  4cpu/16g; 2 nodes: 4cpu/16g
(and other info are welcomed to help us debugging)

**How To Reproduce**
1. deploy any application 

**Expected behavior**

**Comment by zryfish** [comment]
Can you be more specific? It's great if there are step-to-step procedures to help us reproduce this problem. From what I stand now, hard to figure what this problem is?

**Comment by Warden-Thresh** [comment]
> Can you be more specific? It's great if there are step-to-step procedures to help us reproduce this problem. From what I stand now, hard to figure what this problem is?

Deploy any application in app store will case this error，I installed the latest kubesphere 3.0，without any other config changes,I only use this function successful in 2.1.

**Comment by zheng1** [comment]
> > Can you be more specific? It's great if there are step-to-step procedures to help us reproduce this problem. From what I stand now, hard to figure what this problem is?
> 
> Deploy any application in app store will case this error，I installed the latest kubesphere 3.0，without any other config changes,I only use this function successful in 2.1.

I tried it, but this issue cannot be reproduced. Maybe we have fixed it in the latest version.
You can delete the cluster by `kk delete cluster` and execute `docker image prune -a -f` to clean up the docker image cache, and then reinstall the latest version.

**Comment by Warden-Thresh** [comment]
> > > Can you be more specific? It's great if there are step-to-step procedures to help us reproduce this problem. From what I stand now, hard to figure what this problem is?
> > 
> > 
> > Deploy any application in app store will case this error，I installed the latest kubesphere 3.0，without any other config changes,I only use this function successful in 2.1.
> 
> I tried it, but this issue cannot be reproduced. Maybe we have fixed it in the latest version.
> You can delete the cluster by `kk delete cluster` and execute `docker image prune -a -f` to clean up the docker image cache, and then reinstall the latest version.

It works，an solved #2360 👍
